### DISCUSSION TITLE
Version 3.3.2

### DISCUSSION THREAD

**Discussion by jzabroski** [fluentmigrator/fluentmigrator#1529] [Show and tell]
# 💯 Enhancements 
#1528  Add support for Postgres [Insert statement "OVERRIDING { SYSTEM | USER } VALUE"](https://www.postgresql.org/docs/10/sql-insert.html)

# See Also
https://github.com/fluentmigrator/fluentmigrator/milestone/34

<hr /><em>This discussion was created from the release <a href='https://github.com/fluentmigrator/fluentmigrator/releases/tag/v3.3.2'>Version 3.3.2</a>.</em>

**Comment by PhenX** [comment]
Hello, is there any reason this version is not published on nuget yet ?

**Reply by jzabroski** [reply]
@PhenX  I thought it was published.  I will check AzureDevOps now.

**Reply by jzabroski** [reply]
It's on nuget.org now - I forgot to do the approval to release it from Azure to Nuget.  Thanks for raising.
